htdocs/asset/card.php

```diff
@@ -125,38 +125,18 @@
 	include DOL_DOCUMENT_ROOT.'/core/actions_sendmails.inc.php';
```

Comment:
here it's right

Reply:
I means in other document.php from other modules

Reply:
@aspangaro When you declare a file to load it, it is include_once (because declaration must be done once).
When you insert code inside a page (so a file ending with .inc.php), it is include. If you find include_once on a file ending with inc.php, it is a bug.

Reply:
Thanks for your complete answer @eldy . I will proposed a PR for the other situation who have detected